### ISSUE TITLE
MissionFiles Check error (v1.0.2 TBD)

### ISSUE THREAD

**Issue by TheFirstNoob** [BigEgg17/Epoch-Antihack-Admin-Tools#11]
After update **1.0.1 release --> Main (1.0.2 TBD)** i have some spam error (_mic = false stop this problem).
```
22:57:05 Error in expression <ocessFile _x);
			{_arr set [_x, ((_arr select _x) mod 100) + _forEachIndex]} fo>
22:57:05   Error position: <select _x) mod 100) + _forEachIndex]} fo>
22:57:05   Error Zero divisor
```
And 
`22:57:24 Transfer of uninitialized variables is not supported`
On 1.0.1 no errors

**Config.sqf:**
```
//---List of files within your mission. Antihack will make sure that none have been changed.
local _missionFiles =
[ 
"configVariables.sqf","description.ext","init.sqf","loadingscreen.jpg","mission.sqm","rules.sqf","Traders_Menu.sqf","Configs\CfgLoot\Buildings\regex.txt","DZAI_Client\dzai_client_config.sqf","DZAI_Client\dzai_initclient.sqf","Scripts\RndLoadout\RndLoadout.sqf"
];
```

**Comment by looter809** [comment]
I get these too. The "hh:mm:ss Transfer of uninitialized variables is not supported" spams thousands of lines of error too.

**Comment by BigEgg17** [comment]
> After update **1.0.1 release --> Main (1.0.2 TBD)** i have some spam error (_mic = false stop this problem).
> 
> ```
> 22:57:05 Error in expression <ocessFile _x);
> 			{_arr set [_x, ((_arr select _x) mod 100) + _forEachIndex]} fo>
> 22:57:05   Error position: <select _x) mod 100) + _forEachIndex]} fo>
> 22:57:05   Error Zero divisor
> ```
> 
> And
> `22:57:24 Transfer of uninitialized variables is not supported`
> On 1.0.1 no errors
> 
> **Config.sqf:**
> 
> ```
> //---List of files within your mission. Antihack will make sure that none have been changed.
> local _missionFiles =
> [ 
> "configVariables.sqf","description.ext","init.sqf","loadingscreen.jpg","mission.sqm","rules.sqf","Traders_Menu.sqf","Configs\CfgLoot\Buildings\regex.txt","DZAI_Client\dzai_client_config.sqf","DZAI_Client\dzai_initclient.sqf","Scripts\RndLoadout\RndLoadout.sqf"
> ];
> ```

Can you remove ```Configs\CfgLoot\Buildings\regex.txt``` from your mission check array and let me know if the first error (the one preventing execution) is resolved? If so, can you provide that file to me for review.

**Comment by BigEgg17** [comment]
> I get these too. The "hh:mm:ss Transfer of uninitialized variables is not supported" spams thousands of lines of error too.

Does this occur with _mic disabled?

**Comment by TheFirstNoob** [comment]
> > I get these too. The "hh:mm:ss Transfer of uninitialized variables is not supported" spams thousands of lines of error too.
> 
> Does this occur with _mic disabled?

If TRUE - spam errors. If FALSE - all clear work
File can sent later. (I'm not home). On 1.0.1 regex.txt doesn't have problems and work clear on _mic = true. Only 1.0.2 TDB problems
That file u can get on dayz_code/cfgloot.../...

I check all later (after 2-3 days).

**Comment by TheFirstNoob** [comment]
- Can you remove Configs\CfgLoot\Buildings\regex.txt

Fixed on 1.0.2. Remove this file = all work fine thx!

Regex.txt
```
\s*=\s*
 = 
class\s+([_a-zA-Z0-9]+)\s*:\s*([_a-zA-Z0-9]+)\s*\{
class $1 : $2\r\n\{
\{\{
\r\n\t\{\r\n\t\t\{
\}\}
\}\r\n\t\}
^\{([^\r])
\t\t\{$1
([^\n])[ \t]+$
$1
```

**Comment by BigEgg17** [comment]
> * Can you remove Configs\CfgLoot\Buildings\regex.txt
> 
> Fixed on 1.0.2. Remove this file = all work fine thx!
> 
> Regex.txt
> 
> ```
> \s*=\s*
>  = 
> class\s+([_a-zA-Z0-9]+)\s*:\s*([_a-zA-Z0-9]+)\s*\{
> class $1 : $2\r\n\{
> \{\{
> \r\n\t\{\r\n\t\t\{
> \}\}
> \}\r\n\t\}
> ^\{([^\r])
> \t\t\{$1
> ([^\n])[ \t]+$
> $1
> ```

Can you send me your mission file and antihack folder so I can review this? I was able to put the above file into my mission check with no issues.

You can find me on discord under BigEgg#1129

**Comment by TheFirstNoob** [comment]
I reupload all current used files on my git: https://github.com/TheFirstNoob/FireFly-DayZ-Epoch-1.0.7
AH: 1.0.2 and MPMission latest
